README.md

```diff
@@ -191,19 +191,44 @@ CLOUDFLARE_APPLICATION_AUDIENCE=<CLOUFLARE_ACCESS_AUDIENCE_TAG>
 
 ### Docker Build and Run
 
-Build the Docker image
+Build the Docker image and run it using Docker Compose:
```

Comment:
We may want to keep the `docker build -t access .` command in the docks here if people want to build the container standalone without using docker compose.

````suggestion
Build the Docker image:
```
docker build -t access . 
```

Or build and run it using Docker Compose:
````

Reply:
will address this also